run_xnat2bids.py

```diff
@@ -181,6 +184,76 @@ def fetch_job_ids(stdout):
 
     return jobs
 
+def diff_data_directory(bids_root, user, password):
+
+    missing_sessions = []
+
+    # Establish connection 
+    connection = requests.Session()
+    connection.verify = True
+    connection.auth = (user, password)
+
+    host = "https://xnat.bnc.brown.edu"
+
+    # Gather list of existing projects in data directory
+    projects = [proj.name for proj in os.scandir(bids_root) if proj.is_dir()]
```

Comment:
Apologies for an annoying comment, but it might be helpful to have the variable naming here match the related parts of xnat-tools better. So "project" is actually PI, "experiment" is session, etc.

Reply:
No, this is a great point! I'll make the change.